web/index-demo.ejs

```diff
@@ -45,6 +45,15 @@
               app.ports.send.subscribe(sendDataToBlockchain);
               app.ports.get.subscribe(getDataFromBlockchain);
 
+              app.ports.writeLocalStorageImpl.subscribe(function({key, value}) {
+                  localStorage.setItem("sv_" + key, JSON.stringify(value))
+              });
+
+              app.ports.readLocalStorageImpl.subscribe(function(key) {
+                  var ls = localStorage.getItem("sv_" + key)
+                  app.ports.gotLocalStorageImpl.send({key: key, value: JSON.parse(ls)});
+              });
+
```

Comment:
Only improvement I'd make here is a wrapper function to handle errors, something like 

``` javascript
function errWrapper(fName) {
  return function(f) {
    return function(...args){
      try { 
        return f.apply(this, args); 
      } catch (err) {
        // error handling here
        console.error("Function", fName, "produced error:", err);
      }
    }
  }
}

app.ports.readLocalStorageImpl.subscribe(errWrapper("readLocalStorageImpl")(function(key) {
  var ls = localStorage.getItem("sv_" + key)
  app.ports.gotLocalStorageImpl.send({key: key, value: JSON.parse(ls)});
}));
```